lib/_stream_writable.js

```diff
@@ -194,23 +194,16 @@ function writeAfterEnd(stream, cb) {
   process.nextTick(cb, er);
 }
 
-// If we get something that is not a buffer, string, null, or undefined,
-// and we're not in objectMode, then that's an error.
-// Otherwise stream chunks are all considered to be of length=1, and the
-// watermarks determine how many objects to keep in the buffer, rather than
-// how many bytes or characters.
```

Comment:
I removed the comments here because they either duplicated what the inline comments already said or had nothing to do with `validChunk()` ('Otherwise stream chunks ...').